### PR TITLE
Add the simple RDM model

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

**Comment by nomis52** [comment]
This hooks it all together.

File: firmware/src/rdm_responder.c [hunk 1]
```diff
@@ -269,6 +275,62 @@ int RDMResponder_GetSupportedParameters(const RDMHeader *header,
...
+    .dmx_footprint = htons(g_responder.dmx_footprint),
+    .current_personality = g_responder.current_personality,
+    .personality_count = g_responder.personality_count,
+    .dmx_start_address = htons(g_responder.dmx_start_address),
+    .sub_device_count = htons(g_responder.sub_device_count),
+    .sensor_count = g_responder.sensor_count
+  };
+
+  /*
+  // TODO make this .foo format
```
Comment (peternewman) [inline firmware/src/rdm_responder.c]: Haven't you done this above?

Comment (nomis52) [inline firmware/src/rdm_responder.c]: Yup, removed

File: firmware/src/simple_model.h [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ *
+ * simple_model.h
+ * Copyright (C) 2015 Simon Newton
+ */
+
+/**
+ * @defgroup simple_model Simple RDM Model
+ * @brief A simple RDM Model for controlling RGB pixels.
```
Comment (peternewman) [inline firmware/src/simple_model.h]: The simple one isn't doing RGB pixels (yet). Is it worth the simple one just being a straight dummy one anyway.

Comment (nomis52) [inline firmware/src/simple_model.h]: Here's what I'm thinking:
 - 'Simple' (RGB controller)
 - Moving light
 - Sub-device dimmer
 - Sensor only

Each will use the SPI pixels (well, with the exception of sensor only) but the 'simple' one is sort of a native SPI RGB controller device.

File: tests/tests/RDMResponderTest.cpp [hunk 3]
```diff
@@ -34,16 +35,45 @@
 #include "MessageHandlerMock.h"
 
 using ola::rdm::UID;
+/*
```
Comment (peternewman) [inline tests/tests/RDMResponderTest.cpp]: Remove

**Comment by peternewman** [comment]
Generally LGTM, just need the fixes.

File: firmware/src/simple_model.h (hunk 2 above)
Comment (peternewman) [inline firmware/src/simple_model.h]: I think we should have a straight/clean SPI device too though, like the SPI plugin, then one with more features as the simple one. So people can use it for genuine SPI control if they want, without the baggage of fake language setting etc.

File: tests/tests/RDMResponderTest.cpp (hunk 3 above)
Comment (nomis52) [inline tests/tests/RDMResponderTest.cpp]: Done

Comment (peternewman) [inline tests/tests/RDMResponderTest.cpp]: Where @nomis52 ?

**Pull request merged**